resources/eventing/charts/nats-controller/templates/deployment.yaml

```diff
@@ -33,6 +33,16 @@ spec:
             limits:
               cpu: {{ .Values.resources.limits.cpu }}
               memory: {{ .Values.resources.limits.memory }}
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /metrics
```

Comment:
This seems to be a workaround. Can we wait for https://github.com/kyma-project/kyma/issues/9769 to complete?

If this functionality is super urgent then please add a comment to fix this as a part of [this](https://github.com/kyma-project/kyma/issues/9769) issue.